src/test/java/de/tum/in/www1/artemis/architecture/ArchitectureTest.java

```diff
@@ -135,6 +135,21 @@ void testNoCollectorsToList() {
         toListUsage.check(allClasses);
     }
 
+    @Test
+    void testUseKebabCaseForRestEndpoints() {
+        var postMapping = simpleNameAnnotation("PostMapping");
+        var getMapping = simpleNameAnnotation("GetMapping");
+        var putMapping = simpleNameAnnotation("PutMapping");
+        var deleteMapping = simpleNameAnnotation("DeleteMapping");
```

Comment:
The `ResourceArchitectureTest` already contains a list of these endpoint annotations that you can reuse here. I also think that this class would be a better place for this test.

Reply:
Ah true, that's absolutely right :+1: